clio-client/src/main/scala/org/broadinstitute/clio/client/ClioClient.scala

```diff
@@ -18,20 +29,27 @@ object ClioClient extends App {
     ClioClientConfig.ClioServer.clioServerPort,
```

Comment:
Why is `ClioWebClient` created here instead of in the `ClioClient` constructor below?